src/SDKs/ContainerService/Management.ContainerService/Microsoft.Azure.Management.ContainerService.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ContainerService</PackageId>
+    <Description>Provides developers with a library to create and manage all Azure Kubernetes Service resources. </Description>
+    <Version>1.0.0</Version>
+    <AssemblyName>Microsoft.Azure.Management.ContainerService</AssemblyName>
+    <PackageTags>Microsoft Azure Kubernetes Service; Microsoft Azure ContainerService;</PackageTags>
+    <PackageReleaseNotes></PackageReleaseNotes>
```

Comment:
Please add `PackageReleaseNotes`
An example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj#L12)